### PR TITLE
Autodetect mimetype when uploading to bucket

### PR DESCRIPTION
**Pull request by asbjorn**
Hi.

Added a way to autodetect the mimetype when uploading new files to the bucket. content-type was previously not set so all images for instance was using `Content-Type:application/octet-stream`.

### CODE REVIEW

**Comment by Strayer** [comment]
Hey, thank you very much for your contribution! I'll be on vacation next week and can't guarantee to review/merge this before then, but I'll get to it as soon as I can.

**Comment by asbjorn** [comment]
No problem - I've just changed my requirements.txt file to clone my own forked version until this one gets the same capabilities.

If any new changes pops up I'll let you know.

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.3.1
```
Comment (Strayer) [inline setup.cfg]: Please undo this change, since I'd like to handle release bumps myself.

File: django_gcloud_storage/__init__.py [hunk 2]
```diff
@@ -162,9 +163,14 @@ def _save(self, name, content):
 
         # Required for InMemoryUploadedFile objects, as they have no fileno
         total_bytes = None if not hasattr(content, 'size') else content.size
+        content_type = None if not hasattr(content, 'content_type') else content.content_type
+        if content_type is None:
+            _, file_extension = os.path.splitext(name)
+            content_type = mimetypes.types_map.get(file_extension, None)
```
Comment (Strayer) [inline django_gcloud_storage/__init__.py]: This code could be simplified, take a look how django-storages handles it: https://github.com/jschneier/django-storages/blob/f009463/storages/backends/s3boto3.py#L475

I'd prefer:

```python
if content_type is None:
    content_type, _ = mimetypes.guess_type(name)
```

File: django_gcloud_storage/__init__.py [hunk 3]
```diff
@@ -18,7 +19,7 @@
 from google.cloud.exceptions import NotFound
 from google.cloud.storage.bucket import Bucket
 
-__version__ = '0.3.1'
+__version__ = '0.3.2'
```
Comment (Strayer) [inline django_gcloud_storage/__init__.py]: Please undo this change, since I'd like to handle release bumps myself.

**Comment by asbjorn** [comment]
@Strayer Tried to make the adjustments as suggested

**Comment by asbjorn** [comment]
Did you had the chance to review the requested changes @Strayer?

**Comment by Strayer** [comment]
Hey @asbjorn, sorry for taking so horribly long! I just released [0.4.0](https://pypi.org/project/django-gcloud-storage/0.4.0/) on PyPI!

Feel free to get in touch if something is amiss.
